pkg/processor/sumologicschemaprocessor/README.md

```diff
@@ -98,3 +109,42 @@ Below is a list of all attribute keys that are being translated.
 | `k8s.statefulset.name`    | `statefulset`       |
 | `service.name`            | `service`           |
 | `log.file.path_resolved`  | `_sourceName`       |
+
+### Nesting attributes
+
+Nesting attributes allows to change the structure of attributes (both resource attributes and normal ones)
```

Comment:
Aren't resource attributes normal? :smile:
```suggestion
Nesting attributes allows to change the structure of attributes (both resource level and record level attributes)
```

Reply:
They always felt special for me 🙂